src/app/containers/FrontPageSection/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import { bool, shape } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import { StoryPromoUl, StoryPromoLi } from '@bbc/psammead-story-promo-list';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import StoryPromo from '../StoryPromo';
+import groupShape from '../../models/propTypes/frontPageGroup';
+import idSanitiser from '../../lib/utilities/idSanitiser';
+import deepGet from '../../helpers/json/deepGet';
+
+const FrontPageSection = ({ bar, group }) => {
+  const { script } = useContext(ServiceContext);
+  const sectionLabelId = idSanitiser(group.title);
+
+  // The current implementation of SectionLabel *requires* a strapline to be
+  // present in order to render. It is currently *not possible* to render a
+  // section that does not have a strapline without breaking both the visual
+  // *and especially* the screen reader UX.
+  // If this group does not have a strapline; do not render!
+  // This may change in the future, if a way to avoid breaking UX is found.
+  if (!deepGet(['strapline', 'name'], group)) {
+    return null;
+  }
+
+  return (
+    // jsx-a11y considers `role="region"` on a <section> to be redundant.
+    // (<section> tags *should* imply `role="region"`)
+    // While this may be true in a perfect world, we set it in order to get
+    // the greatest possible support.
+    // eslint-disable-next-line jsx-a11y/no-redundant-roles
+    <section role="region" aria-labelledby={sectionLabelId}>
+      <SectionLabel script={script} labelId={sectionLabelId} bar={bar}>
+        {group.strapline.name}
+      </SectionLabel>
+      <StoryPromoUl>
+        {group.items.map(item => (
```

Comment:
I dont think this checks that the key `items` exists. Can you do
```
const items = deepGet(['items'], group);
```
At the top and then throw null if it is falsey (inc empty)